src/util.h

```diff
@@ -281,6 +281,9 @@ try_string_to_int8(const char *nptr, // IN
 /* Disk-resident structures that should be packed are tagged with this. */
 #define ONDISK __attribute__((__packed__))
 
+/* Mnemonic to indicate that a parameter is unused / not-needed */
+#define PARAM_UNUSED 0
```

Comment:
This exists in the platform as `UNUSED_PARAM`

Reply:
Hmmm ... looking closer, that `UNUSED_PARAM` is a compiler directive, `__attribute__((__unused__))`.

What I was adding was a mnemonic for an arg's value as 0, to indicate it's not used by the callee.

In any case, as both names are too close & similar, I withdrew mine. Locally fixed it in the `splinter_test.c`, to use hard-coded value of 0, with a comment.